src/main/java/seedu/commando/logic/commands/UnfinishCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.commando.logic.commands;
+
+import seedu.commando.commons.core.Messages;
+import seedu.commando.commons.exceptions.IllegalValueException;
+import seedu.commando.model.todo.ToDoList;
+import seedu.commando.model.ui.UiToDo;
+import seedu.commando.model.Model;
+import seedu.commando.model.ToDoListChange;
+import seedu.commando.model.todo.ToDo;
+
+import java.util.Optional;
+
+/**
+ * Marks a to-do item as done
```

Comment:
Wrong comment written here, this is the unfinish command, Should be **Marks a finished to-do item as not done** or something like that.